tests/Validator/ValuesOfCorrectTypeTest.php

```diff
@@ -1649,4 +1650,42 @@ public function testListVariablesWithInvalidItem() : void
 
         self::assertTrue($errors[0]->isClientSafe());
     }
+
+    /**
+     * @see it('error messages can be overwriten')
```

Comment:
Is there such a test in the graphl-js reference implementation?

Reply:
Probably no 😁

Reply:
Then this comment can go. They are only added to other tests that directly mirror their implementation.